gulpfile.js

```diff
@@ -51,6 +52,82 @@ const { launch, connect } = require('hadouken-js-adapter');
 		console.log(`[${new Date().toLocaleTimeString()}] ${chalk[color][bgcolor](msg)}.`);
 	}
 
+	/** 
+	* Splits a string version with semantic versioning into an object with major, minor and patch versions
+	* Valid inputs are 'X.X.X' or 'vX.X.X'
+	*/
+	const createSemverObject = (version) => {
+		let tempVersionArray;
+		let semverObject;
+		if (!version || typeof version !== 'string') {
```

Comment:
```suggestion
		if (typeof version !== 'string') {
```

The `!version` check is not necessary because `typeof undefined` is `"undefined"`.  So the `typeof version !== 'string'` is all you need, since it will catch the case where `version` is undefined.